scripts/install

```diff
@@ -42,7 +42,9 @@ focus_user="focus"
 
 videobridge_user="jvb"
 
-max_memory=75
+max_memory=200 #125 mib with no user +1,5*50 users=75 mib
+
+scriptpath=$(pwd)
```

Comment:
instead of `scriptpath=$(pwd)` can't we use:
```
pushd... 

popd
```

Reply:
ok... i learn ;)
i undertand. i will take a lunch and do it ;)

Reply:
you can ping me on the YunoHost Apps Packaging chatroom on matrix if you need it :)